src/leibniz.jl

```diff
@@ -1,20 +1,5 @@
-struct SignVector <: AbstractVector{Float64}
-    len::Int
-end
-Base.size(s::SignVector) = (s.len,)
-Base.getindex(::SignVector, i::Int) = Float64((-1)^iseven(i))
-
-function f(rounds)
-    xs = SignVector(rounds + 2)
-    pi = 1.0
-
-    @simd for i in 2:(rounds + 2)
-        x = xs[i]
-        pi += x / (2 * i - 1)
-    end
-
-    return pi*4
-end
+g(i) = 4 / i
+f(rounds) = sum(g, (-2 * rounds + 1):4:(2 * rounds))
```

Comment:
This is fast for me, and perhaps friendlier for static compilation:
```suggestion
f(n) = sum(i -> 4/(4i-2n-3), 1:n)
```